src/main/java/mate/academy/cinema/controller/InjectDataController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.cinema.controller;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.cinema.model.Role;
+import mate.academy.cinema.security.AuthenticationService;
+import mate.academy.cinema.service.RoleService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class InjectDataController {
+    private final RoleService roleService;
+    private final AuthenticationService authService;
+
+    public InjectDataController(RoleService roleService, AuthenticationService authService) {
+        this.roleService = roleService;
+        this.authService = authService;
+    }
+
+    @PostConstruct
+    public void injectDatabase() {
+        Role user = Role.of("USER");
+        roleService.add(user);
+        Role admin = Role.of("ADMIN");
+        roleService.add(admin);
```

Comment:
```suggestion
        Role admin = roleService.add(Role.of("ADMIN"));
```
давай засетимо так, щоб ми на строці нижче сетили адміну роль, яка вже збережена в базі і з айдішником